### DISCUSSION TITLE
attach query group while running redshift queries

### DISCUSSION THREAD

**Discussion by ramananayak** [sqlalchemy/sqlalchemy#11214] [Usage Questions] [answered]
Redshift offers query_group option to label queries as detailed [here](https://docs.aws.amazon.com/redshift/latest/dg/r_query_group.html).
But when I tried to include this statement it throws an error. So is there any possibility to include this when running redshift queries through sqlalchemy engine

**Comment by zzzeek** [comment]
you would need to ask this over at [sqlalchemy-redshift](https://github.com/sqlalchemy-redshift/sqlalchemy-redshift), we dont maintain redshift here.

**Reply by ramananayak** [reply, marked as answer]
thanks for the response. I have asked the question there. 
https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/issues/301
